src/components/HomePage.vue

```diff
@@ -2,34 +2,56 @@
   import { ref } from 'vue';
   import ToDoList from './ToDoList.vue';
   import HeaderComponent from './HeaderComponent.vue';
-  import toDoListMT from './../assets/toDoListMT.svg';
+  import toDoListPlaceHolder from './../assets/toDoListPlaceHolder.svg';
+  import PopUpToDoListConfirmation from './popUpToDoListConfirmation.vue';
+
   const toDos = ref([]);
+
   function addNewTodo() {
     const newToDo = {
-      title: 'Title',
-      text: 'tanulj Vue.js-t',
+      title: '',
+      text: 'Cras placerat proin.',
```

Comment:
This is not how you implement placeholders, this will make your todo have a base state and if you submit it it will contain these values by default, we do not want that, please remove these hardcoded values, and give proper placeholder text to your inputs